### ISSUE TITLE
Add Pie Chart Helper

### ISSUE THREAD

**Issue by GeorgeBerdovskiy** [PureChart/purechart#15]
## Background
In layman terms, a **pie chart** looks like a circle divided into a number of slices. Each slice has a different color for easy differentiation and takes up some percentage of a larger total (that's where the _pie_ part of the name comes from). Here's an example of a simple pie chart -

![image](https://github.com/PureChart/purechart/assets/43429820/34b85d8d-c833-4081-9651-b880d3a0b297)

Because pie charts are used very often in analytics dashboards, we'd like to add them to PureChart!

## Goal
Create a helper called `pie_chart` in `lib/purechart/chart_helper.rb` that takes a set of data and renders an SVG pie chart using that data. Technically you won't be creating the helper since it already exists (but doesn't do much), so perhaps a more accurate goal would be to _modify_ the existing helper.

This issue is labeled as "advanced" because there some interesting issues to think about. Don't let that intimidate you, though - they are all problems that can be solved with a little creative thinking. For example...
- How do we render _slices_ in SVG format?
- How do we pick colors so that no adjacent slices have the same ones?
  - Should we have a random color generator for this?

Thankfully, others on the internet have thought about some of these issues before. [Here](https://sparkbox.com/foundry/how_to_code_an_SVG_pie_chart) is an article that walks through creating a simple SVG pie chart that you may find helpful.

Good luck 🚀

**Comment by yooian** [comment]
I'll take a crack at the PIE

**Comment by RSK1405** [comment]
Can I also try this one?

**Comment by GeorgeBerdovskiy** [comment]
Sorry @RSK1405, just merged @yooian's PR (#21). However, I'm about to open a number of follow-up issues and will ping you when that's finished.

**Comment by GeorgeBerdovskiy** [comment]
@RSK1405 check out #22, #23, and #24 if interested!
